rfc8999.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: "Version-Independent Properties of QUIC"
 abbrev: QUIC Invariants
-docname: draft-ietf-quic-invariants-latest
-date: {DATE}
+number: 8999
+docName: draft-ietf-quic-invariants-13
+date: 2021-05
```

Comment:
Should this be {DATE} or should {DATE} in draft-ietf-quic-http.md be 2021-05?

Reply:
{DATE} will auto-populate whenever the doc builds, though.  It'll be a fixed value after this, but HTTP doesn't have them yet.

Reply:
Ah that's right!